### ISSUE TITLE
mit-krb5-devel: Unable to compile using LDFLAGS from krb5-config

### ISSUE THREAD

**Issue by cobaugh** [voidlinux/void-packages#7267]
```
$ krb5-config --libs kadm-server
-specs=/void-packages/common/environment/configure/gccspecs/hardened-ld -Wl,-z,relro -Wl,-z,now -Wl,--as-needed -lkadm5srv_mit -lkdb5 -ldb -lgssrpc -lgssapi_krb5 -lkrb5 -lk5crypto -lcom_err
```

krb5-config is just a shell script. One of the vars it sets is:

`LDFLAGS='-specs=/void-packages/common/environment/configure/gccspecs/hardened-ld -Wl,-z,relro -Wl,-z,now -Wl,--as-needed `

**Comment by the-maldridge** [comment]
Sounds like its leaking flags from the build env.  Should be a reasonably simple fix though since it can just be patched out in the post_install().

**Comment by the-maldridge** [comment]
resolved by 230bcf8c29
